From b92c39f0be5552c19923f75aef4487348a08b7dc Mon Sep 17 00:00:00 2001
From: Joe MacDonald <joe_macdonald@mentor.com>
Date: Fri, 20 Feb 2015 21:07:47 -0500
Subject: [PATCH] sandbox: de-bashify

There's no bashisms apparent in either the sandbox initscript nor the
sandboxX script, so point them at /bin/sh instead.

Upstream-Status: Pending

Signed-off-by: Joe MacDonald <joe_macdonald@mentor.com>
Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 sandbox.init | 2 +-
 sandboxX.sh  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sandbox.init b/sandbox.init
index b3979bf..1893dc8 100644
--- a/sandbox.init
+++ b/sandbox.init
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 ## BEGIN INIT INFO
 # Provides: sandbox
 # Default-Start: 3 4 5
diff --git a/sandboxX.sh b/sandboxX.sh
index eaa500d..8755d75 100644
--- a/sandboxX.sh
+++ b/sandboxX.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 trap "" TERM
 context=`id -Z | secon -t -l -P`
 export TITLE="Sandbox $context -- `grep ^#TITLE: ~/.sandboxrc | /usr/bin/cut -b8-80`"
-- 
2.7.4

